src/app/helpers/environmentVariablesHelper.js

```diff
@@ -43,7 +43,8 @@ let envVariables = {
   BUILD_NUMBER: env.build_number || packageObj.version+'.'+packageObj.buildNumber,
   TELEMETRY_SERVICE_LOCAL_URL: env.sunbird_telemetry_service_local_url || 'http://telemetry-service:9001/',
   PORTAL_API_CACHE_TTL: env.sunbird_api_response_cache_ttl || '600',
-  SUNBIRD_EXTCONT_WHITELISTED_DOMAINS: env.sunbird_extcont_whitelisted_domains || 'youtube.com,youtu.be'
+  SUNBIRD_EXTCONT_WHITELISTED_DOMAINS: env.sunbird_extcont_whitelisted_domains || 'youtube.com,youtu.be',
+  TENANT_CDN_URL: env.tenant_cdn_url || ''
```

Comment:
sunbird prefix is required for all environment variable but why we are using another env already we have env for cdn we should use that